hack/common.sh

```diff
@@ -13,3 +13,75 @@ get_operator_sdk() {
   wget -O $DOWNLOAD_DIR https://github.com/operator-framework/operator-sdk/releases/download/v0.18.1/operator-sdk-v0.18.1-x86_64-linux-gnu >/dev/null  && chmod +x /tmp/operator-sdk || return
   echo $DOWNLOAD_DIR
 }
+
+# This function runs operator-sdk run --olm/cleanup depending on the given parameters
+# Parameters:
+# 1: command to run [run/cleanup]
+# 2: path to the operator-sdk binary to use
+# 3: OPTIONAL path to the directory holding the operator manifests
```

Comment:
Isn't this just for the temp CSV with the `image` field replaced? If so then that is what you should say here.